src/pydata_sphinx_theme/__init__.py

```diff
@@ -49,33 +50,59 @@ def update_config(app, env):
         )
 
     # Add an analytics ID to the site if provided
-    # Currently only supports the two types of Google Analytics id.
+    analytics = theme_options.get("analytics", {})
+    # deprecated options for Google Analytics
+    # TODO: deprecate >= v0.12
     gid = theme_options.get("google_analytics_id")
     if gid:
-        # In this case it is "new-style" google analytics
-        if "G-" in gid:
-            gid_js_path = f"https://www.googletagmanager.com/gtag/js?id={gid}"
-            gid_script = f"""
-                window.dataLayer = window.dataLayer || [];
-                function gtag(){{ dataLayer.push(arguments); }}
-                gtag('js', new Date());
-                gtag('config', '{gid}');
-            """
-        # In this case it is "old-style" google analytics
-        else:
-            gid_js_path = "https://www.google-analytics.com/analytics.js"
-            gid_script = f"""
-                window.ga = window.ga || function () {{
-                    (ga.q = ga.q || []).push(arguments) }};
-                ga.l = +new Date;
-                ga('create', '{gid}', 'auto');
-                ga('set', 'anonymizeIp', true);
-                ga('send', 'pageview');
+        msg = (
+            "'google_analytics_id' is deprecated and will be removed in "
```

Comment:
Does this not affect the current Google Analytics section in the documentation?

Reply:
mmm I did not update this indeed.

Reply:
Correction, I mis-read that and I did update the doc about Google Analytics.